src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
-body {
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 100px;
+$calendar-border: $calendar-size / 100px;
```

Comment:
The `$calendar-border` variable should typically be defined in pixels or other fixed units, not as a fraction of another variable like `$calendar-size`. This can lead to unexpected results if the `$calendar-size` changes.